### DISCUSSION TITLE
Hair trigger?

### DISCUSSION THREAD

**Discussion by burntbyhellfire** [sezanzeb/input-remapper#678] [Q&A]
On an earlier version this application, I could easily map a "hair trigger" where even a tiny amount of movement of a trigger sent a BTN_TL2/BTN_TR2 press.  However in the 2.0 version I cannot do this.  I can do a reverse mapping, but then the button is always on unless the trigger is pressed, and threshold doesn't really seem to do anything.  Any ideas?

**Comment by jonasBoss** [comment]
The whole way how Inputremapper does it's magic has changed with the 2.0 release. So it is possible that this is currently not so simple anymore, or it could be a bug.

Can you provide more details?
Could it be that you have the same issue as this: https://github.com/sezanzeb/input-remapper/issues/572#issuecomment-1342436070 ?
